src/app/public/plugin-resources/code-examples/autocomplete/advanced/autocomplete-demo.component.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Component,
+  OnInit
+} from '@angular/core';
+
+import {
+  FormBuilder,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  SkyAutocompleteSearchFunctionFilter,
+  SkyAutocompleteSelectionChange
+} from '@skyux/lookup';
+
+@Component({
+  selector: 'app-autocomplete-demo',
```

Comment:
Is it OK that each of the code samples uses the same selector? I assume so, but just making sure.

Reply:
Yep, they run in isolation, so there's no conflict.